torch/onnx/_internal/fx/op_validation.py

```diff
@@ -195,14 +195,13 @@ def _recursive_wrap_args(
     wrapped_args: List[_type_utils.Argument] = []
     for arg in complete_args:
         if isinstance(arg, torch.fx.Node):
-            fake_tensor = arg.meta["val"]
+            fake_tensor = arg.meta["static_shape"]
             if isinstance(fake_tensor, torch.Tensor):
                 real_tensor = generate_random_tensors(
                     fake_tensor.shape, fake_tensor.dtype
                 )
                 wrapped_args.append(real_tensor)
             elif isinstance(fake_tensor, (int, float)):
-                # TODO(titaiwang): Could dtype be inside a fx.Node?
                 wrapped_args.append(fake_tensor)
             else:
                 warnings.warn(
```

Comment:
Should we raise instead of warn?

Reply:
I don't want op_level_debug to stop exporter. What do you think if we append unsupported types into torch_args, and have another check of args in `validate_op_between_ort_torch` which errors the validation if args are unsupported?

Reply:
So I raise ValueError if it's unsupported types, and wrapped the whole `wrap_fx_args_as_torch_args` with try except ValueError to write the error message into SARIF.